HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -211,6 +211,13 @@ def customiseForOffline(process):
     return process
 
 
+# Remove the explicit CUDAService from the HLT configuration, and
+# rely on ProcessAcceleratorCUDA to load it if necessary
+def customizeHLTfor40725(process):
+    if hasattr(process, 'CUDAService'):
+        del process.CUDAService
+
```

Comment:
There is missing a ```return process`` statement at the end of ```customizeHLTfor40725```

Reply:
🤦🏻‍♂️

Reply:
@Martin-Grunewald thank you so much for catching this !